Create home.html

JavascriptLearner815 opened the pull request.

JavascriptLearner815 committed:
Create home.html

timmy61109 reviewed (commented):
I think you can write some introduction

JavascriptLearner815 commented:
The `<p>Hello World</p>` is a placeholder.

JavascriptLearner815 committed:
Add a bit of content

In this commit I added some content that can be expanded upon and it just stands as a starter.

JavascriptLearner815 committed:
Update home.html

JavascriptLearner815 committed:
Update home.html

JavascriptLearner815 committed:
Create main.css

JavascriptLearner815 committed:
Update home.html

JavascriptLearner815 committed:
Update home.html

JavascriptLearner815 committed:
Add some more CSS magic

timmy61109 reviewed (approved):
It should be possible, although I don’t know the purpose of these
